cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -76,7 +77,8 @@ public MetadataHttpHandler(MetadataAdmin metadataAdmin) {
   public void getAppMetadata(HttpRequest request, HttpResponder responder,
```

Comment:
Should we open a jira to support retrieval of system metadata once we start adding system metadata ?